tool/site/client/src/pages/withdrawPage.tsx

```diff
@@ -17,7 +17,7 @@ import WithdrawForm, {WithdrawFormValues} from "../components/withdraw/withdrawF
 import PrescriptionActions from "../components/common/prescriptionActions"
 import {getArrayTypeGuard, isBundle} from "../fhir/typeGuards"
 import {getMedicationDispenseResources} from "../fhir/bundleResourceFinder"
-import DispenseNotificationsTable from "../components/withdraw/dispenseNotificationsTable"
+import {createPrescriptionDispenseEvents, DispenseEventTable} from "../components/prescription-tracker/dispenseEventsTable/dispenseEventTable"
```

Comment:
since the `DispenseEventTable` component is now used by two pages (withdrawPage and prescription-tracker page) wonder if its worth moving these files to a different location?